crates/consensus/consensus/Cargo.toml

```diff
@@ -16,9 +16,9 @@ reth-primitives.workspace = true
 
 # misc
 auto_impl.workspace = true
-thiserror-no-std = {workspace = true, default-features = false }
+derive_more.workspace = true
 
 [features]
 default = ["std"]
-std = ["thiserror-no-std/std"]
+std = []
```

Comment:
Does it make sense to keep std feature if its array is empty?

Reply:
yes, because other crates can use it to enable `std` behavior in this crate

Reply:
gotcha